### PR TITLE
docs(*): refactor gateway docs 

### PR DESCRIPTION
**Pull request by lukidzi**
Changes to Gateway docs, moved section, changed to the list

### CODE REVIEW

File: docs/docs/dev/explore/gateway.md [hunk 1]
```diff
@@ -1,27 +1,21 @@
 # Gateway
 
-Kuma Gateway is a Kuma component that routes network traffic from outside a Kuma mesh to services inside the mesh.
-The gateway can be thought of as having one foot outside the mesh to receive traffic and one foot inside the mesh to route this external traffic to services inside the mesh.
+Kuma Gateway is responsible for routing network traffic from outside a Kuma mesh to services inside the mesh. Sometimes communication with services outside of the infrastructure is necessary. The gateway ensures that outside traffic reaches the correct service within the Kuma mesh. Also, the gateway itself can handle security at an entrance to the mesh.
```
Comment (lahabana) [inline docs/docs/dev/explore/gateway.md:3]:
```suggestion
When services need to receive traffic from the outside (commonly called North/South) the Kuma Gateway enables routing network traffic from outside a Kuma mesh to services inside the mesh. The gateway is also responsible for security at the entrance of the Mesh.
```

File: docs/docs/dev/explore/gateway.md [hunk 2]
```diff
@@ -1,27 +1,21 @@
...
-Kuma Gateway is deployed as a Kuma `Dataplane`, i.e. an instance of the `kuma-dp` process.
+Kuma Gateway deploys as a Kuma [`Dataplane`](../explore/dpp.md), i.e. an instance of the `kuma-dp` process.
 Like all Kuma `Dataplanes`, the Kuma Gateway `Dataplane` manages an Envoy proxy process that does the actual network traffic proxying.
 
-There exists two types of gateways:
+You can distinguish two types of gateways:
 
-- Delegated: Which enables users to use any existing gateway like [Kong](https://github.com/Kong/kong).
-- Builtin: configures the data plane proxy to expose external listeners to drive traffic inside the mesh.
+- delegated: allows users to use any existing gateway like [Kong](https://github.com/Kong/kong).
```
Comment (lahabana) [inline docs/docs/dev/explore/gateway.md:10]:
```suggestion
- delegated: Allows users to use any existing gateway like [Kong](https://github.com/Kong/kong).
```

File: docs/docs/dev/explore/gateway.md [hunk 3]
```diff
@@ -1,27 +1,21 @@
...
+- builtin: configures the data plane proxy to expose external listeners to drive traffic inside the mesh.
```
Comment (lahabana) [inline docs/docs/dev/explore/gateway.md:11]:
```suggestion
- builtin: Configures the data plane proxy to expose external listeners to drive traffic inside the mesh.
```

File: docs/docs/dev/explore/gateway.md [hunk 4]
```diff
@@ -109,7 +104,7 @@ spec:
 " | kubectl apply -f -
 ```
 
-You can access your ingress with `curl -i $PROXY_IP/foo` where `$PROXY_IP` is the ip retrieved from the service that exposes Kong outside your cluster.
+You can access your ingress with `curl -i $PROXY_IP/foo` where `$PROXY_IP` you can retrieve from the service that exposes Kong outside your cluster.
```
Comment (lahabana) [inline docs/docs/dev/explore/gateway.md:107]:
```suggestion
You can access your ingress with `curl -i $PROXY_IP/foo` where `$PROXY_IP` is retrieved from the service that exposes Kong outside your cluster.
```

**Review by lahabana — APPROVED** [review]

**Comment by slonka** [comment]
@lukidzi - can you merge this if there is no more work on your side?

**Pull request merged**